pkg/controllers/mcs/service_export_controller.go

```diff
@@ -435,6 +450,8 @@ func (c *ServiceExportController) removeOrphanWork(ctx context.Context, endpoint
 		workName := names.GenerateWorkName(endpointSlice.GetKind(), endpointSlice.GetName(), endpointSlice.GetNamespace())
 		willReportWorks.Insert(workName)
 	}
+	klog.V(5).Infof("Will remove endpointslice(%s/%s) doesn't under namespace %s",
+		serviceExportKey.Namespace, serviceExportKey.Name, names.GenerateExecutionSpaceName(serviceExportKey.Cluster))
```

Comment:
I'm confused by this log.
1. Are you going to remove the EndpointSlice or the Work that holds the EndpointSlice?
2. Why are you going to remove the EndpointSlice anyway?

Reply:
You're right; the log description here is unclear. What we need to delete is ` the Work that holds the EndpointSlice`.